pkg/sources/s3/s3.go

```diff
@@ -457,7 +457,7 @@ func (s *Source) pageChunker(
 
 		// Ignore large files.
 		if *obj.Size > s.maxObjectSize {
-			ctx.Logger().V(5).Info("Skipping %d byte file (over maxObjectSize limit)")
+			ctx.Logger().V(5).Info("Skipping large file", "key", *obj.Key, "size", *obj.Size, "max_object_size", s.maxObjectSize)
```

Comment:
note: I believe the key, and size should already be in the [context](https://github.com/trufflesecurity/trufflehog/pull/3835/files#diff-dd074dd61c256804b0485d58de413da6ba4589eb889e5da850cadf3040f0d8fdR442)

Reply:
aah. good catch Ahrav!